integreat_compass/cms/views/offers/offer_report_view.py

```diff
@@ -0,0 +1,87 @@
+from django.contrib import messages
+from django.shortcuts import redirect, render
+from django.utils.translation import gettext as _
+from django.views.generic import TemplateView
+
+from ...forms import ReportForm
+from ...models import Offer, OfferVersion
+
+
+class OfferReportView(TemplateView):
+    """
+    View to report an offer
+    """
+
+    template_name = "offers/offer_report.html"
+
+    @staticmethod
+    def _get_form():
+        """
+        This function gets the form Report
+
+        :return: the report form
+        :rtype: dict
+        """
+        form = ReportForm()
+        return form
+
```

Comment:
I think this was copied and modified from the `OfferFormView`, correct? There we used this method because we had to initialize so many forms both in GET and POST methods. It's not necessary here (thankfully)
```suggestion
```